dask_geopandas/tests/test_core.py

```diff
@@ -833,3 +833,19 @@ def test_to_dask_dataframe(geodf_points_crs):
     assert isinstance(result, pd.DataFrame) and not isinstance(
         result, geopandas.GeoDataFrame
     )
+
+
+def test_compute_empty_partitions():
+    # https://github.com/geopandas/dask-geopandas/issues/190 - ensure to skip
+    # empty partitions when concatting the computed results
```

Comment:
This test is based on the report from https://github.com/geopandas/dask-geopandas/issues/190, but it is maybe not a fully "correct" test, in that it (accidentally, because of pandas defaulting to float64 for empty data) creates a somewhat "incorrect" dask GeoDataFrame (the first empty partition has dtypes not matching `meta`). 

But it _is_ a test that currently fails on main, and passes with this PR.